docker/package/model.py

```diff
@@ -451,7 +451,7 @@ class TezosBakingServicesPackage(AbstractPackage):
     # native releases, so we append an extra letter to the version of
     # the package.
     # This should be reset to "" whenever the native version is bumped.
-    letter_version = "a"
+    letter_version = ""
```

Comment:
Am I wrong or this could be updated to `b` instead of the `release` be bumped to `2`?

Reply:
What for is the release number at all then? AFAIU, `letter_version` is specifically for updating `tezos_baking` (i.e. package's source e.g. `tezos-setup-wizard`), whereas this will really affect every package, so we should bump release number instead.

However, it's not really documented anywhere :shrug: , so I might be wrong.

Reply:
`letter_version` is required only for `tezos_baking` because we tend to updated sources of this package (i.e. `tezos_setup_wizard.py`), while the other _normal_ packages rely on the sources gathered via `opam-bundle` and they shouldn't change in scope of one upstream version.

I see that the latest published version of tezos-baking is `2:9.4a-0ubuntu1`. However, AFAIU, we've changed the sources of the `tezos_setup_wizard.py` since that, so `letter_version` has to be updated for the newly published version of `tezos-baking` package

Reply:
> This should be reset to "" whenever the native version is bumped.

Note that upstream version wasn't bumped, so it should be `letter_version = "b"` instead